analytics/article_impressions.js

```diff
@@ -38,10 +38,34 @@ if(location.pathname.match('/article/') || location.pathname.match('/articles'))
               id: 'magazine_fixed'
             }
           }
+        }else if(classList.contains('article-es-form')){
+          var article = $(this).parent().parent().parent().parent();
```

Comment:
Yep! Definitely read the article -- it's an interesting comparison between `parents` and `closest` and does some performance testing to prove that `closest` is the faster choice, but this works too!